spec/feature/authenticated_user_spec.rb

```diff
@@ -121,16 +121,43 @@
 
   it "checkout" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
-    and_return(valid_user)
+                                                    and_return(valid_user)
     click_add_to_cart_link("Breakfast")
     click_link_or_button "Cart:"
     click_link_or_button "Checkout"
+    save_and_open_page
     within("#flash_notice") do
       expect(page).to have_content("Your delicious food is on the way")
     end
   end
 
-  xit "can view their past orders with links to each order" do
+  it "can view their past orders with links to each order" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(valid_user)
+    click_add_to_cart_link("Breakfast")
+    click_link_or_button "Cart:"
+    click_link_or_button "Checkout"
+    within('#item-title') do
+      expect(page).to have_content("Bacon and Eggs")
+    end
+    within('#item-description') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.